code_of_conduct.md

```diff
@@ -1,57 +1,78 @@
+# Code of Conduct
+
 PyCon US is a community conference intended for networking and collaboration in the developer community.
 
-We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by PyCon or not.
+We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by the Python Software Foundation or not.
 
 To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and volunteers at any PyCon event are required to conform to the following Code of Conduct. Organizers will enforce this code throughout the event.
 
-The Short Version
------------------
+## Our Standards
 
-PyCon is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of conference participants in any form.
+PyCon is dedicated to providing a harassment-free conference experience for everyone, regardless of age, gender identity and expression, sexual orientation, disability, physical appearance, body size, ethnicity, nationality, race, or religion (or lack thereof), education, or socio-economic status.
 
-All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any conference venue, including talks.
+Examples of behavior that contributes to creating a positive environment include:
```

Comment:
@sagesharp Do we have any good source material for adding a few more points to this list? It feels a tad imbalanced against the following list.

Reply:
Sure! I only left in the positive examples that were already in the document.

The [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) has some short social guidelines:
 * Using welcoming and inclusive language
 * Being respectful of differing viewpoints and experiences
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members

The [Django Project](https://www.djangoproject.com/conduct/) has some more detailed social guidelines, but most of them are pretty well summarized by the above points.

The [Recurse Center](https://www.recurse.com/code-of-conduct) has a set of social rules that are geared towards being very newcomer-friendly:
 * **No feigning surprise.** The first rule means you shouldn't act surprised when people say they don't know something. This applies to both technical things ("What?! I can't believe you don't know what the stack is!") and non-technical things ("You don't know who RMS is?!"). Feigning surprise has absolutely no social or educational benefit: When people feign surprise, it's usually to make them feel better about themselves and others feel worse. And even when that's not the intention, it's almost always the effect. As you've probably already guessed, this rule is tightly coupled to our belief in the importance of people feeling comfortable saying "I don't know" and "I don't understand."
 * **No well-actually's.** A well-actually happens when someone says something that's almost - but not entirely - correct, and you say, "well, actually…" and then give a minor correction. This is especially annoying when the correction has no bearing on the actual conversation. This doesn't mean we aren't interested in truth-seeking or that we don't care about being precise. Almost all well-actually's in our experience are about grandstanding, not truth-seeking. (Thanks to Miguel de Icaza for originally coining the term "well-actually.")

Depending on how newcomer-friendly you want the PyCon sprints to be, you might consider adopting the two rules from the Recurse Center. I would encourage adopting the bullet points from the Contributor Covenant at the very least.

Reply:
Thank you @sagesharp! The Recurse Center social rules are of my new favorites (I recently attended a batch). I think they are _strongly_ effective, but am concerned that they do take a little bit more introduction than we can reasonably allot.

I'll add the Contributor Covenant points, and we could consider the Recurse Center's social rules for inclusion in time.